imcsdk/apis/server/remotepresence.py

```diff
@@ -24,13 +24,28 @@
 from imcsdk.mometa.comm.CommVMediaMap import CommVMediaMap
 from imcsdk.mometa.sol.SolIf import SolIf, SolIfConsts
 from imcsdk.imcexception import ImcOperationError
-from imcsdk.imccoreutils import get_server_dn
+from imcsdk.imccoreutils import get_server_dn, IMC_PLATFORM
 CIFS_URI_PATTERN = re.compile('^//\d+\.\d+\.\d+\.\d+\/')
 NFS_URI_PATTERN = re.compile('^\d+\.\d+\.\d+\.\d+\:\/')
 
 
+def _get_kvm_parent_dn(handle, server_id):
```

Comment:
this seems to be same as `_get_ipmi_parent_dn`. why have 2 copies and not import?